.github/workflows/build.yml

```diff
@@ -98,11 +104,19 @@ jobs:
       - name: Restore build artifact permissions
         run: cd dist && setfacl --restore=permissions-backup.acl
         continue-on-error: true
-      - name: Prepare Repository
-        run: mv dist .repo
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          ref: ${{ github.event.pull_request.head.ref }}
```

Comment:
Can't we use `PULL_REQUEST_REF` from above? if not, I guess we can remove it there and inline it like done here